src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime/Microsoft.Azure.CognitiveServices.LUIS.Runtime.csproj

```diff
@@ -6,15 +6,14 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime</PackageId>
     <Description>Provides API functions for consuming the Microsoft Azure Cognitive Services LUIS Runtime API.</Description>
-    <Version>2.1.0</Version>
+    <Version>2.8.0-preview</Version>
```

Comment:
Please update the assembly version in `AssemblyInfo.cs` and also the `PackageReleaseNotes` here